scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -273,8 +273,7 @@
                    'Useful for testing the benchmark script, as this allows '
                    'distributed mode to be run on a single machine. For '
                    'example, if there are two tasks, each can be allocated '
-                   '~40 percent of the memory on a single machine',
-                   lower_bound=0., upper_bound=1.)
+                   '~40 percent of the memory on a single machine')
```

Comment:
Add a sentence to the end of the help string saying something like "This is also useful for using unified memory, as this can be set above 1 to oversubscribe the GPU using unified memory."